CRVReco/src/CrvRecoPulsesFinder_module.cc

```diff
@@ -167,9 +170,8 @@ namespace mu2e
       const CRSScintillatorBarIndex &barIndex = digi.GetScintillatorBarIndex();
       int SiPM = digi.GetSiPMNumber();
       uint16_t startTDC = digi.GetStartTDC();
-      std::vector<int16_t> ADCs;
+      std::vector<int16_t> ADCs=digi.GetADCs();
```

Comment:
Can you access the ADCs from the digi as const&? That would avoid an unnecessary copy.

Reply:
Yes, GetADCs() returns the digis as const&. I'm going to change this part of the code and also look for other places.

Reply:
It turns out that this vector can't be a const, because subsequent digis may be added for longer pulses.